chain-test/src/test/scala/org/bitcoins/chain/blockchain/ChainHandlerTest.scala

```diff
@@ -10,22 +10,21 @@ import org.bitcoins.core.number.{Int32, UInt32}
 import org.bitcoins.core.p2p.CompactFilterMessage
 import org.bitcoins.core.protocol.BlockStamp
 import org.bitcoins.core.protocol.blockchain.BlockHeader
-import org.bitcoins.core.util.{FutureUtil, TimeUtil}
+import org.bitcoins.core.util.{TimeUtil}
```

Comment:
I think having a single item with `{}` will cause an error, should remove them, doing ctrl + alt + o, will automatically do this. Could be worth running optimize imports on the whole project if we are doing a cleanup